src/test/docker/docker-run-checks.sh

```diff
@@ -42,20 +42,22 @@ Options:\n\
      --inception               Run tests as flux jobs\n\
      --system                  Run under system instance\n\
  -q, --quiet                   Add --quiet to docker-build\n\
- -t, --tag=TAG                 If checks succeed, tag image as NAME\n\
- -i, --image=NAME              Use base docker image NAME (default=$IMAGE)\n\
- -p, --platform=NAME           Run on alternate platform (if supported)\n\
- -S, --flux-security-version=N Install flux-security vers N (default=$FLUX_SECURITY_VERSION)\n
- -j, --jobs=N                  Value for make -j (default=$JOBS)\n
+ -t, --tag TAG                 If checks succeed, tag image as NAME\n\
```

Comment:
Thanks!

The commit message should be updated to use the `Problem:` statement form and wrapped at 72 chars.

Maybe we should move the test:
```sh
# check if running in OSX
if [[ "$(uname)" == "Darwin" ]]; then
    # BSD getopt
```

to above where the help string is generated and make the use of `=` and MacOS warning conditional on that? 

Another idea would be to detect BSD getopt and suggest installing GNU getopt instead.

These are just suggestions besides the commit message changes.